scripts/check-asn.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+cat ./asn1/*.asn |
+	# Convert POST body to CRLF
+	sed $'s/$/\r/' |
+	# "@-" means input is from stdin.
+	# "%24" is a URL-encoded "$" (curl expects the field name already escaped).
+	curl --silent -X POST \
+		--data-urlencode '__EVENTTARGET=' \
+		--data-urlencode '__EVENTARGUMENT=' \
+		--data-urlencode '__LASTFOCUS=' \
+		--data-urlencode '__VIEWSTATE=/wEPDwUJOTIzNTU1MjUxD2QWAmYPZBYCAgMPZBYCAgEPZBYKAgEPEGQPFgkCAgIDAgQCBQIGAgcCCAIJAgoWCRAFDklUUyBDQU0gdjEuMy4yBQ5JVFMgQ0FNIHYxLjMuMmcQBQ9JVFMgREVOTSB2MS4yLjIFD0lUUyBERU5NIHYxLjIuMmcQBQlJVFMgSjI3MzUFCUlUUyBKMjczNWcQBQtOQkFQIHY2LjIuMAULTkJBUCB2Ni4yLjBnEAUMUkFOQVAgVjYuMi4wBQxSQU5BUCBWNi4yLjBnEAUKUlJDIHY4LjYuMAUKUlJDIHY4LjYuMGcQBQxTMUFQIHY4LjEwLjAFDFMxQVAgdjguMTAuMGcQBQhUQVAtMDMxMAUIVEFQLTAzMTBnEAULWDJBUCB2OC45LjAFC1gyQVAgdjguOS4wZxYBZmQCDQ8QZGQWAWZkAiEPEGRkFgFmZAIjD2QWAgIBDxBkZBYBZmQCKQ8PFgIeBFRleHQFWnsNCiAgIm5hbWUiOiJBZGFtIiwNCiAgImZpcnN0LXdvcmRzIjoiSGVsbG8gV29ybGQiLA0KICAiYWdlIjp7DQogICAgImJpYmxpY2FsIjo5MzANCiAgfQ0KfWRkGAEFHl9fQ29udHJvbHNSZXF1aXJlUG9zdEJhY2tLZXlfXxYBBR1jdGwwMCRNYWluQ29udGVudCRjYk5vUmVsYXhlZIkssErSHkM+Q9I95FYQz7JtxQKe+/7KUwMgkm90gRoO' \
```

Comment:
Will this viewstate work indefinitely? Might need a step to fetch a fresh one each time.